src/components/withLocalize.js

```diff
@@ -87,6 +96,15 @@ class LocaleContextProvider extends React.Component {
         return NumberFormatUtils.format(this.props.preferredLocale, number, options);
     }
 
+    /**
+     * @param {Number} number
+     * @param {Intl.NumberFormatOptions} options
+     * @returns {Array}
+     */
+    formatToParts(number, options) {
+        return NumberFormatUtils.formatToParts(this.props.preferredLocale, number, options);
+    }
+
```

Comment:
Do we need this for this PR?

Reply:
We are using it in `isCurrencySymbolLTR`

Reply:
I don't think we are.

Reply:
This will be used in a newly created `isCurrencySymbolLTR`

Reply:
I am still not sure. Could you please tell me the line number and filename?

Reply:
```javascript

    isCurrencySymbolLTR(currencyCode) {
        const parts = this.props.formatToParts(0, {
            style: 'currency',
            currency: currencyCode,
        });

        // The first element of parts will be type: currency for all currency
        // Where it starts with symbol and the other will have it at last
        // If it is not the first, it must be at last
        const isLeft = parts[0].type === 'currency';
        return isLeft;
    }

```

Reply:
But current code is 
```
 const parts = NumberFormatUtils.formatToParts(this.props.preferredLocale, 0, {
            style: 'currency',
            currency: currencyCode,
        });
```

Also, we don't have to create this function here. It just wrapping `NumberFormatUtils.formatToParts`. You can call that directly.

Reply:
Yes, we can directly call that but I followed the already implemented pattern like `numberFormat` exists in `withLocale` that handles passing `prefferedLocale`.